index.js

```diff
@@ -117,4 +117,46 @@ app.post('/payments/create', async (request, result) => {
     }
 })
 
+// Endpoint used to edit customer
+app.post('/customers/edit', async (request, result) => {
+    const customerData = request.body;
+    console.log("Customer Edit Payload: ", customerData);
+
+    try {
+        const { name, tariffPlanId, siteId, meterNumber, hasmeter, isactive, id, customerUpdate } = customerData;
+        const { dateUpdated, customerIds, explanation, userId } = customerUpdate;
+        const airtableCustomerData = {
+            isactive,
+            hasmeter,
+            tariffPlanId,
+            siteId,
+            name,
+            meterNumber,
+        };
+        const airtableCustomerUpdate = {
+            dateUpdated,
+            customerIds,
+            explanation,
+            userId: userId[0]
+        };
+
+        await updateCustomer(id, airtableCustomerData);
+        console.log("Customer edited!");
+
+        result.status(201);
+        result.json({ status: 'OK' })
```

Comment:
I think we only need to send 1 JSON status (this is supposed to indicate the API request was successful). Let's delete this 1 and just leave the bottom 1